project/app.py

```diff
@@ -1,14 +1,23 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, abort, flash, render_template, request, redirect, url_for
+from config import Config
 from Database.database_setup import db, User, Channel, UserChannel
 from channel_management import create_channel, delete_channel
 from utils import get_current_user_id, is_user_channel_admin
+from flask_login import LoginManager, login_required, login_user, logout_user
+from forms import RegisterForm, LoginForm
 
 app = Flask(__name__, static_url_path='/static')
+app.config.from_object(Config)
+
+login_manager = LoginManager()
+login_manager.init_app(app)
+login_manager.login_view = "login"
+
 
 # Configuration for SQLAlchemy
-app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://mango:COSC310=mcpw@127.0.0.1/mangochat'
+# app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://mango:COSC310=mcpw@127.0.0.1/mangochat'
```

Comment:
Is there a reason all of this is commented out?

Reply:
I moved the config to another file and left that there as an artifact. I can remove it.

Reply:
Okay cool, that makes sense.  No need to remove it.

Reply:
In the name of refactoring, I have removed it.